malaria-ios/ViewControllers/InfoHub/AchievementProperty.swift

```diff
@@ -0,0 +1,31 @@
+import Foundation
+import CoreData
+
+/// One of the conditions so that an achievement can be unlocked.
+
+class AchievementProperty: NSManagedObject {
+  
+  enum ActivationMethod: String {
```

Comment:
Move everything that isn't automatic generated by the datamodel to a separate file AchivementPropertyExtension. If you regenerate the class you will lose everything.